pkg/kubelet/cm/devicemanager/endpoint.go

```diff
@@ -31,7 +31,7 @@ import (
 // device states reported by the device plugin.
 type endpoint interface {
 	getPreferredAllocation(available, mustInclude []string, size int) (*pluginapi.PreferredAllocationResponse, error)
```

Comment:
if memory serves this one could use a context as well

Reply:
yes, added context to this one